image/oci/layout/oci_dest_test.go

```diff
@@ -216,3 +217,136 @@ func TestPutblobFromLocalFile(t *testing.T) {
 	err = ociDest.CommitWithOptions(context.Background(), private.CommitOptions{})
 	require.NoError(t, err)
 }
+
+// TestPutSignaturesWithFormat tests that sigstore signatures are properly stored in OCI layout
+func TestPutSignaturesWithFormat(t *testing.T) {
```

Comment:
(Absolutely non-blocking: consider consolidating the tests into a table-driven one. This is perfectly fine as is.)